measure_extinction/utils/plot_bandpasses.py

```diff
@@ -28,5 +28,5 @@ def plot_bandpasses(bands):
     band_path = pkg_resources.resource_filename("measure_extinction", "data/Band_RespCurves/")
 
     # define the different bandpasses
-    bands = ["JohnU", "JohnB", "JohnV", "JohnR", "JohnI", "JohnJ", "2MASSJ", "JohnH", "2MASSH", "JohnK", "JohnKs", "2MASSKs", "AAOL", "IRAC1", "AAOLprime", "IRAC2", "AAOM", "IRAC3", "IRAC4"]
+    bands = ["JohnU", "JohnB", "JohnV", "JohnR", "JohnI", "JohnJ", "2MASSJ", "JohnH", "2MASSH", "JohnK", "JohnKs", "2MASSKs", "AAOL", "IRAC1", "AAOLprime", "IRAC2", "AAOM"]
```

Comment:
What happened to IRAC3 and IRAC4?

Reply:
Not used for SpeX?

Reply:
They are at larger wavelengths and the plot becomes unclear if you include so many bands, so I limited to those relevant for SpeX (and lower). But you can add/remove any band that you need basically.

Reply:
Cool.  Thanks for the info.